pkg/plugin/trivy/plugin.go

```diff
@@ -1134,8 +1136,12 @@ func (p *plugin) getPodSpecForStandaloneFSMode(ctx starboard.PluginContext, conf
 			},
 		})
 	}
-
-	return corev1.PodSpec{
+	if ctx.GetStarboardConfig().IsScanJobsInSameNamespace() {
+		for i, _ := range containers {
```

Comment:
Do we need this loop? Maybe we can determine image pull policy in the beginning of the method and just set it when we create containers?